backend/router/data.py

```diff
@@ -64,16 +65,21 @@ def cluster_data(confirmed, clusters_config=None):
         df["confirmed"], nb_class=clusters_config["clusters"]
     )
 
+    rounded_breaks = list(map(lambda limit: round(limit, sigfigs=3), breaks))
```

Comment:
100% open to discussion, but, do you really think it's worth it to add another lib to the backend when the front can do this out of the box?

Reply:
I add it in the back since
 * at first was considering this rounded limits for clustering the data, then realized that it was not a good solution for what i mentioned in the PR description
* I like the fact that front doesn't make decisions about the data itself, it just formats it in the way it likes.

Reply:
and I don't feel like rounding the data is just format ..